src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -923,6 +923,289 @@ public void SarifTransformerTests_ToVersionOne_NotificationExceptionWithStack()
             VerifyCurrentToVersionOneTransformation(V2LogText, V1LogExpectedText);
         }
 
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_ResultLocations()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner""
+      },
+      ""logicalLocations"": {
+        ""html.body.header"": {
+          ""name"": ""header"",
+          ""decoratedName"": ""!!PageHeader#asdf"",
+          ""kind"": ""member"",
+          ""parentKey"": ""html.body""
+        },
+        ""html.body"": {
+          ""name"": ""body"",
+          ""kind"": ""namespace"",
+          ""parentKey"": ""html""
+        },
+        ""html"": {
+          ""name"": ""html"",
+          ""kind"": ""namespace""
+        }
+      },
+      ""results"": [
+        {
+          ""ruleId"": ""WEB1079"",
+          ""message"": {
+            ""arguments"": [
+              ""shape""
+            ]
+          },
+          ""ruleMessageId"": ""default"",
+          ""locations"": [
+            {
+              ""physicalLocation"": {
+                ""fileLocation"": {
+                  ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                },
+                ""region"": {
+                  ""startLine"": 20,
+                  ""startColumn"": 69,
+                  ""endColumn"": 74,
+                  ""offset"": 720,
```

Comment:
Of course this will change when the region PR is merged.

Reply:
Yep